blog/_posts/2018-04-27-concepts-at-work.md

```diff
@@ -0,0 +1,65 @@
+
+## Data Collection and Core Data Set
+
+At an early stage of her investigation, Karin put together a list of main actors and publications in the research on MIG. This list was based on a growing collection of original papers as well as the present historiography which includes many recollections of participating astronomers. Based on the citations and references of this first selection of papers which were provided by the online data bases Web of Science (WoS) and ADS, this selection grew into a snow ball sample. Combined keyword searches added further pertinent papers. The ADS soon proved to be the better data base especially since its search engine is open source and hence allows customized access to its data. Web of Science, by contrast, is now owned by private equity investors Onex and Baring Asia - and according to Chadegani (2013) a subscription to WoS might cost a larger institution such as an university around 100,000 dollars a year.
+
+The open source search engine of the ADS soon came in handy. Karin's assistant Theodor Costea, who only learnt how to code in Python during the project, developed a script using the ADS API to extract the pertaining references and citations from the ADS into the open source DB browser for SQLite (<http://sqlitebrowser.org/>). The script allows not only a compound search including authors names, title and keywords as well as time intervals, but also defines the network type (e.g. co-citation or bibliographic coupling) and adds weighted attributes. A recent addition is the community detecting algorithm *InfoMap* (©<http://mapequation.org>) in the script. This algorithm takes into account the directionality of a direct citation network, as opposed to, say "modularity" (Blondel, 2008) which is integrated into the open source network visualizing software gephi (<https://gephi.org/>). Theo's script will soon be published on Github (<https://github.com/>) so that many more scholars or people simply interested in astrophysics research dynamics might profit from it.
```

Comment:
Add a link here for the ADS API (http://adsabs.github.io/help/api/ or https://github.com/adsabs/adsabs-dev-api). If Theodor used Andy Casey's Python client, it'd be good to link to that too (https://github.com/andycasey/ads)

Reply:
I think there's also a typo in the sentence ending with gephi. "... which is integrated into" I think refers to InfoMap, but seems to refer to "modularity" in the sentence. Can you double check?